### PR TITLE
john-jumbo 1.9.0

### PR DESCRIPTION
**Pull request by hidde-jan**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by equal-l2** [comment]
@fxcoudert Will you re-run CI, please?

**Comment by SMillerDev** [comment]
@brewtestbot test this please

**Comment by SMillerDev** [comment]
```
==> make -s CC=clang
ar: creating archive aes.a
ar: creating archive secp256k1.a
ar: creating archive ed25519-donna.a
Undefined symbols for architecture x86_64:
  "_fmt_stribog_256", referenced from:
      _john_register_all in john.o
  "_fmt_stribog_512", referenced from:
      _john_register_all in john.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [../run/john] Error 1
make: *** [default] Error 2
```

File: Formula/john-jumbo.rb [hunk 1]
```diff
@@ -22,23 +22,6 @@ class JohnJumbo < Formula
...
-  # upstream.  See
-  # https://github.com/magnumripper/JohnTheRipper/issues/1901
-  patch do
-    url "https://github.com/magnumripper/JohnTheRipper/commit/d29ad8aabaa9726eb08f440001c37611fa072e0c.diff?full_index=1"
-    sha256 "b3400f54c64dccce6fe4846872c945b280ec221c7a3d614b03c18029cba3695a"
-  end
-
   def install
     cd "src" do
       system "./configure", "--disable-native-tests", "--disable-native-macro"
```
Comment (equal-l2) [inline Formula/john-jumbo.rb]: `--disable-native-macro` has renamed to `--disable-native-march`.

**Comment by equal-l2** [comment]
- `README` at root directory has gone. Remove L.33.
- `john.ini` won't be accepted anymore. Remove from L.43 to L.45

**Comment by equal-l2** [comment]
Instead of the patch removed, JOHN_SYSTEMWIDE families should be configured through CFLAGS:

```
 ENV.append "CFLAGS", "-DJOHN_SYSTEMWIDE=1"
 ENV.append "CFLAGS", "-DJOHN_SYSTEMWIDE_EXEC=\\\"#{prefix}/share/john\\\""
 ENV.append "CFLAGS", "-DJOHN_SYSTEMWIDE_HOME=\\\"#{prefix}/share/john\\\""
```

**Pull request closed without merging**